### ISSUE TITLE
Charts axis ticks are not completely visible

### ISSUE THREAD

**Issue by imrankhurram-checkit** [jqPlot/jqPlot#208]
Charts axis ticks are not completely visible as something is drawn on the canvas over the ticks. It looks like this is a bug in Primefaces 12 having jqplot but is there any workaround? Below are samples:
![image](https://github.com/jqPlot/jqPlot/assets/64078961/9b6c4a55-8b29-4a0d-a75c-ef2076ec7e46)
![image (2)](https://github.com/jqPlot/jqPlot/assets/64078961/3758c4ad-55f8-423b-8c61-690dcda69559)

**Comment by harbulot** [comment]
Is that on Chrome/Edge with a Intel graphics?

It looks like it's the same problem as #204 (with a suggested fix in #205).

**Comment by imrankhurram-checkit** [comment]
@harbulot yes it is on chrome and edge. Can you suggest me some steps to use this patch in Primefaces 12 ?

**Comment by srikanth2k15** [comment]
@harbulot, Any update for ticks rendering issue fix?
